assets/js/base/components/title/style.scss

```diff
@@ -1,12 +1,12 @@
 // Extra class for specificity to overwrite editor styles.
-.wc-block-component__title.wc-block-component__title {
+.wc-block-components-title.wc-block-components-title {
 	@include reset-box();
 	@include font-size(large);
 }
 
 // For Twenty Twenty we need to increase specificity a bit more.
 .theme-twentytwenty {
-	.wc-block-component__title.wc-block-component__title {
```

Comment:
I see what you mean but I still think it's better to have theme-specific styles at the end of the files. It causes some duplication, but makes it easy to know where to find them and if at some point we consider moving those styles to specific files for each theme (ie: `twentytwenty.scss`, `twentynineteen.scss`, etc.) it will be easier to find them if they are always at the same place.

Reply:
Thanks for explaining, I can see the benefits of keeping the theme overrides in one place :)